### PR TITLE
add fabric8-analytics-server base

### PR DESCRIPTION
**Pull request by syamgk**

### CODE REVIEW

File: base/fabric8-analytics-server/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM registry.devshift.net/osio-prod/base/pcp:latest
+
+COPY hack/_copr_jpopelka-mercator.repo /etc/yum.repos.d/
+
+# python3-pycurl is needed for Amazon SQS (boto lib), we need CentOS' rpm - installing it from pip results in NSS errors
+RUN yum-config-manager --enablerepo=rhel-7-server-optional-rpms && \
```
Comment (jmelis) [inline base/fabric8-analytics-server/Dockerfile]: For what package is this required?

Comment (syamgk) [reply base/fabric8-analytics-server/Dockerfile]: mercator requires maven which requires this

Comment (jmelis) [reply base/fabric8-analytics-server/Dockerfile]: :+1:

**Comment by jmelis** [comment]
MERGED

**Pull request closed without merging**